virtualization/windowscontainers/docker/optimize_windows_dockerfile.md

```diff
@@ -73,7 +73,7 @@ bd6c831b55b8        2 minutes ago       cmd /S /C powershell.exe -command Remove
 91508fd744e5        3 minutes ago       cmd /S /C powershell.exe -command Invoke-WebR   51.92 MB
 6801d964fda5        5 months ago
 ```
-To compare, here is the same operation, however all steps run with the same RUN instruction. Note that each step in the RUN instruction is on a new line of the Dockerfile, the '/' character is being used to line wrap. 
+To compare, here is the same operation, however all steps run with the same RUN instruction. Note that each step in the RUN instruction is on a new line of the Dockerfile, the `/` character is being used to line wrap. 
```

Comment:
It's actually a backslash that is the line continuation character, not `/`. But looking at this example, it's actually a really bad one. See Dockerfile.windows in the docker/docker master branch source code for a better way of doing multiple PS commands in one `RUN` (but you don't need the download function it uses). Powershell needs to be set to stop if a command fails, otherwise docker won't know what failed. Let's say in this example below, there's a typo in that URL and it doesn't exist. Docker will see the error from the Remove-Item, not the Invoke-WebRequest. Worse, let's say you didn't have the `Remove-Item`, the `RUN` would apparently succeed on the URL typo case.  Then there's the next issue - rather than running the exe and sleeping, use `Start-Process -wait file.exe -ArgumentList "params"`.